src/components/messenger/MessageBox.vue

```diff
@@ -184,14 +210,15 @@ export default {
         this.msgFromMe = true;
         this.msgSender = this.activeUser[0].username;
         this.msgReceiver = this.routeData.car_id.user_id.username;
-        return `Deine Buchungsanfrage an ${this.routeData.car_id.user_id.username}`;
+        return `Deine Buchungsanfrage an ${this.routeData.car_id.user_id.username} (${this.routeData.car_id.user_id.firstname} ${this.routeData.car_id.user_id.lastname})`;
       } else if (
         this.activeUser[0].username !== this.routeData.user_id.username
       ) {
         this.msgFromMe = false;
-        this.msgSender = this.routeData.car_id.user_id.username;
+        //this.msgSender = this.routeData.car_id.user_id.username;
+        this.msgSender = this.routeData.user_id.username;
         this.msgReceiver = this.activeUser[0].username;
-        return `${this.routeData.user_id.username} hat dir eine Buchungsanfrage geschickt.`;
+        return `${this.routeData.user_id.username} (${this.routeData.user_id.firstname} ${this.routeData.user_id.lastname}) hat dir eine Buchungsanfrage geschickt.`;
```

Comment:
Bitte hier den zusätzlichen Vor- und Nachnamen entfernen. Der Username reicht eigentlich aus. Und zu viel Infos sollen auch nicht in die Karten.
Zudem befinden sich die Infos ja auch schon im Title.